src/app/Components/Details/RentTransactions/index.jsx

```diff
@@ -1,21 +1,32 @@
 import React, { Component } from 'react';
+import './index.scss';
 
 export default class RentTransactions extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      transactionsDisplayed: 10
+      transactionsDisplayed: 20,
```

Comment:
I'd just make this a const at the top of the file because it's not state it's configuration